rest_framework/templates/rest_framework/admin.html

```diff
@@ -247,7 +247,7 @@ <h4 class="modal-title" id="myModalLabel">{{ error_title }}</h4>
         <script>
           window.drf = {
             csrfHeaderName: "{{ csrf_header_name|default:'X-CSRFToken' }}",
-            csrfCookieName: "{{ csrf_cookie_name|default:'csrftoken' }}"
+            csrfToken: "{{ csrf_token }}"
```

Comment:
Again `{% csrf_token %}`? And do we need the `if request` check?

Reply:
We do need that check, because there is currently a unit test for rendering with no request and in that case the csrf_token call throws an exception (see https://code.djangoproject.com/ticket/29785, which you closed as "by design").

Reply:
OK, read the comments out of order. Here we *don't* need the `{% if request %}` check because (presumably) there will always be a request when you load an admin page. In the `base.html` case, however, the check is only there because of the exception thrown if there is no request to satisfy `test_templates.test_base_template_with_no_context()`.